mods/default/furnace.lua

```diff
@@ -251,6 +253,8 @@ local function furnace_node_timer(pos, elapsed)
 	local infotext
 	if active then
 		infotext = S("Furnace active")
+        -- Play sound while the furnace is active
+		minetest.sound_play("default_furnace_active", {pos = pos, max_hear_distance = 16, gain = 0.1})
```

Comment:
Please document the sound files: author and license in default's `README.txt`.

Reply:
Done.